src/devices/sound/rf5c400.cpp

```diff
@@ -5,16 +5,6 @@
 
     Written by Ville Linde
     Improvements by the hoot development team
-
-    history -
-    2007-02-08 hoot development team
-        looping
-        stereo panning
-        8-bit sample support
-
-    2007-02-16 hoot development team
-        envelope
-        fixed volume table
```

Comment:
Removed these because I saw another PR where it was mentioned changelogs shouldn't be included in the source code. Not sure if this applies to really old code, but this changelog information doesn't really provide much.